--- gcc-4.7.3.orig/gcc/cp/cfns.gperf	2009-04-21 12:03:23.000000000 -0700
+++ gcc-4.7.3/gcc/cp/cfns.gperf	2018-04-29 10:48:16.281353226 -0700
@@ -23,6 +23,9 @@
 #ifdef __GNUC__
 __inline
 #endif
+#ifdef __GNUC_STDC_INLINE__
+__attribute__ ((__gnu_inline__))
+#endif
 const char * libc_name_p (const char *, unsigned int);
 %}
 %%
--- gcc-4.7.3.orig/gcc/cp/cfns.h	2009-04-21 12:03:23.000000000 -0700
+++ gcc-4.7.3/gcc/cp/cfns.h	2018-04-29 10:48:16.281353226 -0700
@@ -54,6 +54,9 @@
 #ifdef __GNUC__
 __inline
 #endif
+#ifdef __GNUC_STDC_INLINE__
+__attribute__ ((__gnu_inline__))
+#endif
 const char * libc_name_p (const char *, unsigned int);
 /* maximum key range = 391, duplicates = 0 */
 

